src/drivers/px4fmu/fmu.cpp

```diff
@@ -1026,74 +1026,75 @@ PX4FMU::cycle()
 			}
 		}
 
-		/* can we mix? */
-		if (_mixers != nullptr) {
+	}
 
-			size_t num_outputs;
+	_cycle_timestamp = hrt_absolute_time();
```

Comment:
ah, I didn't realize the cause was the timestamp not being updated, maybe the mixing code should go back in the if block